### PR TITLE
Added A New Project (Get_Time_Timezones)

### PR DESCRIPTION
**Pull request by codingis4noobs2**
With The Help Of Get_Time_TimeZones Project You Can Fetch Time From Different Timezones :)

### CODE REVIEW

File: Get_Time_TimezoneWise/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,159 @@
+absl-py==0.13.0
```
Comment (hastagAB) [inline Get_Time_TimezoneWise/requirements.txt:1]: Why there are so many libraries in `requirements.txt` ?

Comment (hastagAB) [inline Get_Time_TimezoneWise/requirements.txt:1]: Why there are so many libraries in `requirements.txt` ?

**Review by hastagAB — CHANGES_REQUESTED** [review]

File: Get_Time_TimezoneWise/requirements.txt (hunk 1 above)
Comment (codingis4noobs2) [reply Get_Time_TimezoneWise/requirements.txt]: Updated requirements.txt

**Comment by codingis4noobs2** [comment]
Done @hastagAB :)

**Review by hastagAB — APPROVED** [review]

**Pull request merged**